Backend/Rooms/Service/RoomService.cs

```diff
@@ -39,6 +48,16 @@ public Room GetRoomByRoomNumber(int roomNumber)
             }
             return null;
         }
+
+        public List<Room> GetRoomsByRoomLabel(string roomLabel)
+        {
+            return _context.Rooms.ToList().Where(p => p.RoomLabel.ToLower().Contains(roomLabel)).ToList();
+        }
+        public List<Room> GetRoomsByRoomUse(string roomUse)
+        {
+            return _context.Rooms.ToList().Where(p => p.RoomUse.ToLower().Contains(roomUse)).ToList();
+        }
+
         public Room UpdateRoom(Room room) => roomRepository.Update(room);
```

Comment:
when writing these kind of functions place => expression in new line like in SIMS videos from last year